upgrading/blue_green_deployments.adoc

```diff
@@ -210,58 +207,55 @@ projects should be isolated and pods in global projects should be able to access
 ----
 
 [[blue-green-warming-green-nodes]]
-== Warming the Green Nodes
+== Preparing the Green Nodes
 
-In order for pods to be migrated from the blue environment to the green, the
-required container images must be pulled. Network latency and load on the
-registry can cause delays if there is not sufficient capacity built in to the
-environment.
+To migrate pods from the blue environment to the green, you must pull the
+required container images. Network latency and load on the registry can cause
+delays if the environment does not have sufficient capacity.
```

Comment:
@kalexand-rh These two sentences don't work together for me. They go from something the user does to a system concern. 
_Make sure you have sufficient  capacity to avoid any issues with network latency and load._ 

Or move the sentence as is to the next paragraph?